app/parsers/bulkrax/csv_parser.rb

```diff
@@ -136,7 +132,12 @@ def create_new_entries
       current_work_ids.each_with_index do |wid, index|
         break if limit_reached?(limit, index)
         new_entry = find_or_create_entry(entry_class, wid, 'Bulkrax::Exporter')
-        Bulkrax::ExportWorkJob.perform_now(new_entry.id, current_run.id)
+        entry = Bulkrax::ExportWorkJob.perform_now(new_entry.id, current_run.id)
+
+        if entry
+          self.headers ||= []
+          self.headers |= entry.parsed_metadata.keys
```

Comment:
I think this code is wrong - the headers should be the "from"s not the "to"s

Reply:
turns out this is right, its the headers that csv entry send out that were wrong.